src/Moryx.Products.Management/ModuleController/ModuleController.cs

```diff
@@ -66,6 +66,7 @@ protected override void OnInitialize()
 
             // Load all product plugins
             Container.LoadComponents<IProductStorage>();
+            Container.LoadComponents<IConfiguredTypesProvider>();
```

Comment:
Load components is not necessary for internal components, only for extensible plugins

Reply:
Ohh, good to know 😅